TMDBApp/TMDBApp/ListMovieViewController.swift

```diff
@@ -0,0 +1,177 @@
+//
+//  ViewController.swift
+//  TMDBApp
+//
+//  Created by Richard Leh on 17/12/2016.
+//  Copyright © 2016 Richard Leh. All rights reserved.
+//
+
+import UIKit
+import Alamofire
+import SVProgressHUD
+
+enum jsonResult:String {
+    case results    = "results"
+    case genres     = "genres"
+    case totalPages = "total_pages"
+}
+
+class ListMovieViewController: UIViewController {
+
+    @IBOutlet weak var tableView:UITableView!
+    let tableCellIdentifier:String = "MovieCell"
```

Comment:
Determine the type of variable by value passed.
In this case I could just write `let tableCellIdentifier = "MovieCell"`